pubsub/appengine/data/privatekey.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Public keys are fine to upload, but should private keys be okay to post publicly? Are there any security implications this might bring?

In other words, is this safe testing data that would only be used by this sample and cannot be used by any other means whatsoever?

Reply:
Confirmed with the team this is just a sample/dummy which is part of the original sample. Need to fix the regional tag too tho.

Closing this PR for now as we decided to approach this differently rather than duplicating it. Thank you for your review